tokens/color/status.yaml

```diff
@@ -0,0 +1,95 @@
+color:
+  status:
+    $extensions:
+      com.redhat.ux:
+        order: 500
+
+    danger:
+      on-light:
+        $value: '{color.red-orange.50}'
+        $description: Danger accent color (light theme)
+      on-dark:
+        $value: '{color.red-orange.40}'
+        $description: Danger accent color (dark theme)
+      surface:
+        on-light:
+          $value: '{color.red-orange.10}'
+          $description: Danger surface color (light theme)
+        on-dark:
+          $value: '{color.red-orange.70}'
+          $description: Danger surface color (dark theme)
+
+    warning:
+      on-light:
+        $value: '{color.orange.50}'
+        $description: Warning accent color (light theme)
+      on-dark:
+        $value: '{color.orange.40}'
+        $description: Warning accent color (dark theme)
+      surface:
+        on-light:
+          $value: '{color.orange.10}'
+          $description: Warning surface color (light theme)
+        on-dark:
+          $value: '{color.orange.70}'
+          $description: Warning surface color (dark theme)
+
+    caution:
+      on-light:
+        $value: '{color.yellow.60}'
+        $description: Caution accent color (light theme)
+      on-dark:
+        $value: '{color.yellow.30}'
+        $description: Caution accent color (dark theme)
+      surface:
+        on-light:
+          $value: '{color.yellow.10}'
+          $description: Caution surface color (light theme)
+        on-dark:
+          $value: '{color.yellow.70}'
+          $description: Caution surface color (dark theme)
+
+    neutral:
+      on-light:
+        $value: '{color.gray.60}'
+        $description: Neutral accent color (light theme)
+      on-dark:
+        $value: '{color.gray.40}'
+        $description: Neutral accent color (dark theme)
+      surface:
+        on-light:
+          $value: '{color.gray.10}'
+          $description: Neutral surface color (light theme)
+        on-dark:
+          $value: '{color.gray.80}'
+          $description: Neutral surface color (dark theme)
+
+    note:
```

Comment:
these names don't exactly line up with the names already in use in `<rh-alert>`. are these names set in stone from on high?

https://ux.redhat.com/elements/alert/code/

Reply:
@bennypowers Marionne and I are using `Alert` to see how we can start applying the new the colors to elements with statuses, so no, not from on high, but we are aiming for a hardened convention that can be token-ized across more elements.

Reply:
@bennypowers, I was using [this Figma file](https://www.figma.com/file/1YkKJofXZGfjHHt2t8ajWn/2023-color-palette?type=design&node-id=0-1&mode=design&t=qTwUV92lfGmomwhR-0) from Brand/DPO to name the different status levels. 

The only alert variant name that will have to change is "info," and "caution" and "neutral" will have to be added. Colors will have to change, but hopefully this will be an improvement because `<rh-alert>` is using crayon names right now.

Reply:
these are the alert states. changing them would break backwards


- default - Indicates generic information or a message with no severity.
- info - Indicates helpful information or a message with very little to no severity.
- success - Indicates a success state, like if a process was completed without errors.
- warning - Indicates a caution state, like a non-blocking error that might need to be fixed.
- danger - Indicates a danger state, like an error that is blocking a user from completing a task.

Reply:
Would we want to tackle `<rh-alert>` changes before merging status tokens then?

Reply:
We will likely have to update `<rh-badge>` as well. It uses the following states: info, success, moderate, important, critical.

Reply:
Alert has first mover advantage here, and imo it behooves us to carefully weigh that advantage.

Tokens can adopt alert's nomenclature without issue, because tokens as-of-yet have no concept of state.

Or token can innovate a new nomenclature. But then: either alert is retroactively rendered odd, or tokens are seen as overly abstract.

At that point there are two options (assuming tokens are authoritative), we can either break alert (and every instance of it, worldwide); or we can assert that alert has its own private language for state, then privately map alert states to token states

-----

I prefer for tokens to adopt alert states, because alert exists today whereas token states are still hypothetical

Reply:
@bennypowers Excellent insight as always.

@marionnegp Does this help us move forward?

Reply:
@coreyvickery, yep! Thanks, @bennypowers. I also checked with Lucia from the PatternFly team, and they're planning on keeping their alert names the same. Since `<rh-alert>`'s names are already aligned with PatternFly's, I think we can go ahead and model the status token names after alert.